src/util/autoclose.ts

```diff
@@ -39,18 +45,18 @@ export class AutoClose {
                                  filter(e => e.which === Key.Escape));
 
 
-        // we have to pre-calculate 'shouldCloseOnClick' on 'mousedown',
-        // because on 'mouseup' DOM nodes might be detached
-        const mouseDowns$ =
-            fromEvent<MouseEvent>(this._document, 'mousedown').pipe(map(shouldCloseOnClick), takeUntil(closed$));
+        // we have to pre-calculate 'shouldCloseOnClick' on 'mousedown/touchstart',
+        // because on 'mouseup/touchend' DOM nodes might be detached
+        const mouseDowns$ = fromEvent<MouseEvent>(this._document, iOS ? 'touchstart' : 'mousedown')
+                                .pipe(map(shouldCloseOnClick), takeUntil(closed$));
 
-        const outsideClicks$ = fromEvent<MouseEvent>(this._document, 'mouseup')
-                                   .pipe(
-                                       withLatestFrom(mouseDowns$), filter(([_, shouldClose]) => shouldClose), delay(0),
-                                       takeUntil(closed$));
+        const closeableClicks$ = fromEvent<MouseEvent>(this._document, iOS ? 'touchend' : 'mouseup')
+                                     .pipe(
+                                         withLatestFrom(mouseDowns$), filter(([_, shouldClose]) => shouldClose),
+                                         delay(iOS ? 16 : 0), takeUntil(closed$));
```

Comment:
If anybody has a better solution for this, please share it :)

Problem is when there is a `click` handler on a triggering element (say, button): 
- to open: `click` → `open()` 
- to close: `touchdown` → `touchup` → `click (is still generated that leads to)` → `open() (popup reopening)`

by delaying the `touchup`, `click` and `open()` are processed while the popup is still open